pokemongo_bot/__init__.py

```diff
@@ -1021,63 +947,32 @@ def update_web_location_worker(self):
             self.web_update_queue.get()
             self.update_web_location()
 
-    def get_inventory_count(self, what):
-        response_dict = self.get_inventory()
-        inventory_items = response_dict.get('responses', {}).get('GET_INVENTORY', {}).get(
-            'inventory_delta', {}).get('inventory_items', {})
-        if inventory_items:
-            pokecount = 0
-            itemcount = 1
-            for item in inventory_items:
-                if 'inventory_item_data' in item:
-                    if 'pokemon_data' in item['inventory_item_data']:
-                        pokecount += 1
-                    itemcount += item['inventory_item_data'].get('item', {}).get('count', 0)
-        if 'pokemon' in what:
-            return pokecount
-        if 'item' in what:
-            return itemcount
-        return '0'
-
-    def get_player_info(self):
-        response_dict = self.get_inventory()
-        inventory_items = response_dict.get('responses', {}).get('GET_INVENTORY', {}).get(
-            'inventory_delta', {}).get('inventory_items', {})
-        if inventory_items:
-            pokecount = 0
-            itemcount = 1
-            for item in inventory_items:
-                # print('item {}'.format(item))
-                playerdata = item.get('inventory_item_data', {}).get('player_stats')
-                if playerdata:
-                    nextlvlxp = (int(playerdata.get('next_level_xp', 0)) - int(playerdata.get('experience', 0)))
-
-                    if 'level' in playerdata and 'experience' in playerdata:
-                        self.logger.info(
-                            'Level: {level}'.format(
-                                **playerdata) +
-                            ' (Next Level: {} XP)'.format(
-                                nextlvlxp) +
-                            ' (Total: {experience} XP)'
-                            ''.format(**playerdata))
-
-                    if 'pokemons_captured' in playerdata and 'poke_stop_visits' in playerdata:
-                        self.logger.info(
-                            'Pokemon Captured: '
-                            '{pokemons_captured}'.format(
-                                **playerdata) +
-                            ' | Pokestops Visited: '
-                            '{poke_stop_visits}'.format(
-                                **playerdata))
-
-    def has_space_for_loot(self):
-        number_of_things_gained_by_stop = 5
-        enough_space = (
-            self.get_inventory_count('item') <
-            self._player['max_item_storage'] - number_of_things_gained_by_stop
-        )
-
-        return enough_space
+    # def get_player_info(self):
```

Comment:
Yep we do. But I deleted the api call. So we can’t access to player stats for now.
These stats are used in the UpdateLiveStats. Gotta find a better way than having a method. Since it’s coupled with the inventory datas we should add a class which make the api call an handle the instantiation of the inventory if needed as well as facilitate the access to player infos.